yogstation/code/modules/mob/living/carbon/human/species_types/halloween.dm

```diff
@@ -0,0 +1,5 @@
+/datum/species/shadow/ling/check_roundstart_eligible()
```

Comment:
this should be moved to the sling species folder, which is in the sling gamemode folder

Reply:
it's not tho